src/patch-evaluation-service/pyproject.toml

```
@@ -0,0 +1,12 @@
+[project]
+name = 'autopatch.patch-evaluation-service'
+version = '0.1.0'
```

```suggestion
version = '0.2.3'
```
suggest aligning to current release version